docs/source/resources/developer_guide.rst

```diff
@@ -0,0 +1,254 @@
+===============
+Developer Guide
+===============
+
+This is the developer guide for Superstaq. If you have questions, please email superstaq@infleqtion.com or reach out on our `Slack workspace <https://join.slack.com/t/superstaq/shared_invite/zt-1wr6eok5j-fMwB7dPEWGG~5S474xGhxw>`_.
+
+.. contents:: **Table of Contents**
+   :depth: 3
+   :local:
+   :backlinks: none
+
+Developer Workflows
+===================
+
+Installation
+------------
+Please visit our `README <https://github.com/Infleqtion/client-superstaq/blob/main/README.md#installation-for-development>`_ for developer installation instructions.
+
+Modifying Superstaq
+-------------------
+
+Push Edits and Make a Pull Request for Review
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+#. Update the main branch (e.g., ``git pull``) and update relevant dependencies (``python3 -m pip install -e .``, ``python3 -m pip install <general-superstaq, qiskit-superstaq, cirq-superstaq. --upgrade``).
```

Comment:
this install line isn't quite right - should be something like,
```suggestion
#. Update the main branch (e.g., ``git pull``)
#. Install/update relevant dependencies. To work on a single package, navigate to relevant subdirectory (e.g. `cd cirq-superstaq`) and run ``python3 -m pip install -e .``. Alternatively, install all ``client-superstaq`` packages by navigating to the repository root and running ``python3 -m pip install -e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq -e ./cirq-superstaq -e ./supermarq-benchmarks`` (in that order).
```
(though probably cleaned up a bit)

Reply:
Add the ^ language with some formatting - let me know how it looks and if you want me to update anything!

Reply:
perfect, thanks!